src/main/java/com/endava/soa_5/page_objects/FilterByPricePage.java

```diff
@@ -0,0 +1,26 @@
+package com.endava.soa_5.page_objects;
+
+import com.endava.soa_5.abstract_classes.BaseClass;
+import lombok.Getter;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+@Getter
+public class FilterByPricePage extends BaseClass {
+    @FindBy(xpath = "//div[@class='page-title']//h1")
+    private WebElement booksPageTitle;
+    @FindBy(xpath = "//div[@class='filter-title']//strong")
+    private WebElement filterByPriceTitle;
+    @FindBy(xpath = "//a[contains(@href,'http://demowebshop.tricentis.com/books?price=-25')]")
+    private WebElement priceUnder25FilterButton;
+    @FindBy(xpath = "//a[contains(@href,'http://demowebshop.tricentis.com/books?price=25-50')]")
+    private WebElement priceBetween25And50FilterButton;
+    @FindBy(xpath = "//a[contains(@href,'http://demowebshop.tricentis.com/books?price=50')]")
+    private WebElement priceOver50FilterButton;
```

Comment:
if possible, try to use the shorter XPath, it's easy to maintain for a longer period.
ex: //a[contains(@href,'/books?price=-25')]

Reply:
Done.